core/drivers/fabric-driver/server/fabric-code.ts

```diff
@@ -161,28 +159,37 @@ async function invoke(
 
         // 4. Prepare the view and return.
         const viewPayload = new view_data.FabricView();
-        const endorsements: proposalResponse.Endorsement[] = [];
+        const endorsedProposalResponses: view_data.FabricView.EndorsedProposalResponse[] = [];
         //TODO Fix ts error
         //@ts-ignore
+        let ii = 0;
```

Comment:
If we are going to leave this in the code, rename this to be more descriptive, like `endorsementCounter`.

Reply:
Yeah I thought for debugging purpose would be good to have those logs for future